client/src/components/FilesDialog/FilesDialog.vue

```diff
@@ -3,10 +3,24 @@
         :error-message="errorMessage"
         :options-show="optionsShow"
         :modal-show="modalShow"
-        :hide-modal="() => (modalShow = false)">
+        :hide-modal="() => (modalShow = false)"
+        :back-func="() => load()"
```

Comment:
I believe it is considered better practice to emit an event from the child instead of passing callbacks to the child if possible :)

Reply:
Yes, but I just wanted to follow code style (like `hide-modal`). So, I changed it.